src/main/java/flamegrapher/model/Processes.java

```diff
@@ -3,30 +3,28 @@
 import java.util.HashMap;
 import java.util.Map;
 
-import com.fasterxml.jackson.annotation.JsonProperty;
-
 public class Processes {
     
-    @JsonProperty("processes")
-    Map<String, String> pidName;
+    Map<String, Item> items;
     
     public Processes() {
-        pidName = new HashMap<String,String>();
+        items = new HashMap<String,Item>();
     }
     
-    public Map<String, String> getPidName() {
-        return pidName;
+    public Map<String, Item> items() {
+        return items;
     }
 
     public static Processes fromString(String string) {
         Processes p = new Processes();
         String[] lines = string.split("\n");
         for(String line : lines) {
             String[] parts = line.split(" ", 2);
-            p.pidName.put(parts[0], parts[1]);
+            String pid = parts[0];
```

Comment:
how about using .split(System.lineSeparator()) in line 20 to make sure it works on Windows